ansible/roles/jenkins-master/meta/main.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 dependencies:
-  - role: handlers-jenkins-master
+  - role: nginx
```

Comment:
`jenkins-seed-job` and the role recently created `jenkins-pipeline` do not have this file with `handlers-jenkins-master` as a dependency. I'll fix it in my PR, please remind me to update it if yours is merged first.